src/EnergyPlus/CurveManager.cc

```diff
@@ -2088,11 +2087,19 @@ namespace CurveManager {
                         ErrorsFound = true;
                     } else if (pointsSpecified) {
                         // normalizeGridValues normalizes to 1.0 at the reference values. We must redivide by passing in the 1.0/normalizationDivisor as the scalar here.
```

Comment:
Comment should probably be updated to match.

Reply:
Hi Julien. Can you elaborate on this? The above comment on line 2090 is new, I added it to help explain what is happening with the line below it. Hopefully it helps clarify the math behind multiplying by the normalizationDivsor inside of the calculation to reset it.

Reply:
I'm seeing the comment as existing:

![image](https://user-images.githubusercontent.com/5479063/74164772-81e4a380-4c24-11ea-8b4a-b6ecbd725393.png)


Also, before it was indeed passing 1.0/normalizationDivisor below, now it does not.

![image](https://user-images.githubusercontent.com/5479063/74164844-9c1e8180-4c24-11ea-8a22-ba84c2c04184.png)

Reply:
You are correct, I thought that I had changed this comment, but it appears that it did not make it in. I'm sorry for the confusion, thanks for clarifying. It is updated to be more descriptive now.